packages/css-library/src/stylesheets/utilities.scss

```diff
@@ -24,5 +24,136 @@ $utilities-package: map-collect(
   $u-width
 );
 
-// Generate all utility classes
+// Generate all utility classes from USWDS function
 @include render-utilities-in($utilities-package);
+
+// Generate medium screen breakpoint utility classes.
+// If this USWDS PR gets approved/merged, we can remove all the code below:
+// https://github.com/uswds/uswds/pull/5985
+$medium-screen-breakpoint: 768px;
+$breakpoint-class-name: 'medium-screen';
+
+@mixin generate-utilities-from-settings($collection) {
+  @each $property, $property-map in $collection { 
+    $u-base: map-get($property-map, base);
+    $u-values: map-get($property-map, values);
+    $u-property: map-get($property-map, property);
+    $u-modifiers: map-get($property-map, modifiers);
+
+    @if $u-modifiers {
+      @each $modifier, $modifier-value in $u-modifiers {
+        @each $name, $value in $u-values {
+          .#{$breakpoint-class-name}\:#{$u-base}#{$modifier-value}-#{$name} {
+            @if $modifier == 'noModifier' {
+              #{$u-property}: #{$value};
+            } @else {
+              #{$u-property}#{$modifier-value}: #{$value};
+            }
+          }
+        }
+      }
+    }
+
+    @each $name, $value in $u-values {
+      .#{$breakpoint-class-name}\:#{$u-base}-#{$name} {
+        #{$u-property}: #{$value};
+      }
+    }
+  }
+}
+
+@media all and (min-width: $medium-screen-breakpoint) {
+  // $u-border
+  @include generate-utilities-from-settings($u-border);
+
+  // $u-display
+  @include generate-utilities-from-settings($u-display);
+
+  // $u-flex-grid
+  @each $property, $property-map in $u-flex-grid { 
+    $u-base: map-get($property-map, base);
+    $u-values: map-get($property-map, values);
+    $u-property: map-get($property-map, property);
+
+    @each $name, $value in $u-values {
+      $extend: map-get($value, extend);
```

Comment:
I don't understand what `$extend` ends up evaluating to here. Could you explain?

Reply:
The `flex-grid` utilities have a block of CSS styles that are named `extend` in the tokens like this (maybe not the greatest name) :

https://github.com/department-of-veterans-affairs/component-library/blob/3ca3fcc424f932a70feda6f871fa6847b56739cd/packages/css-library/src/tokens/flex-grid.scss#L10-L18

I wasn't able to just dump this into the CSS brackets (maybe my Sass inexperience?) so I'm iterating over them instead like this:

```
  @each $css-property, $css-value in $extend {
    #{$css-property}: $css-value;
  }
```

With the flex property added in the end, it generates like this:

```css
  .medium-screen\:vads-l-col {
    box-sizing: border-box;
    flex-basis: 0;
    max-width: 100%;
    min-height: 1px;
    min-width: 0;
    position: relative;
    width: 100%;
    flex-grow: 1;
  }
```